routes/folders.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const express = require('express');
+const mongoose = require('mongoose');
+
+const Folder = require('../models/folder');
+const Note = require('../models/note');
+
+const router = express.Router();
+
+router.get('/', (req, res, next) => {
+  Folder.find()
+    .sort({ name: 'asc' })
```

Comment:
ascending is the default

```suggestion
    .sort('name')
```